### PR TITLE
Feat/filter in search

### PR DESCRIPTION
**Pull request by Jephuff**
Not sure of a good way to indicate that "status: Pending" will filter status.
Placeholder text `e.g. "status: Pending" or "Phil"`?

### CODE REVIEW

File: src/services/login.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ import ajax from 'services/ajax';
 const gapi = global.gapi;
 gapi.load('auth2');
 
-export const googleLogin = token =>
-  ajax.get('/oauth/google/token', { headers: { token } });
+export const googleLogin = idToken =>
+  ajax.get('/oauth/google/token', { headers: { id_token: idToken } });
```
Comment (cheapsteak) [inline src/services/login.ts]: Are you still pointing to the old API? this needs to be `token`

```
REACT_APP_API=https://demo.ego.kfdrc.org/
```

File: src/services/login.ts [hunk 2]
```diff
@@ -16,8 +16,8 @@ export const googleLogout = () => {
   }
 };
 
-export const facebookLogin = token =>
-  ajax.get('/oauth/facebook/token', { headers: { token } });
+export const facebookLogin = idToken =>
+  ajax.get('/oauth/facebook/token', { headers: { id_token: idToken } });
```
Comment (cheapsteak) [inline src/services/login.ts]: ditto ^

File: src/stateProviders/provideList.ts [hunk 3]
```diff
@@ -30,7 +30,18 @@ export default provideState({
 
     refreshList: async effects => {
       const { list: { params, type } } = await effects.getState();
-      const response = await RESOURCE_MAP[type].getList(params);
+      const match = (params.query || '').match(/^(.*)status:\s*("([^"]*)"|([^\s]+))(.*)$/);
+      var [, before, , statusQuoted, statusUnquoted, after] = match || Array(5);
```
Comment (cheapsteak) [inline src/stateProviders/provideList.ts]: Is the `var` significant here?

**Comment by cheapsteak** [comment]
Good question re: indicator

Leave it alone for now, we can add that later once we figure it out

**Comment by cheapsteak** [comment]
Good to merge once the `token`s are reverted back

File: src/services/login.ts (hunk 1 above)
Comment (Jephuff) [reply src/services/login.ts]: ahh yes, I must be.

**Pull request merged**